### PR TITLE
Fix tag extraction

### PR DESCRIPTION
**Pull request by patrickkwang**

### CODE REVIEW

File: src/output.js [hunk 1]
```diff
@@ -49,11 +49,11 @@ class Captor {
         // console.log(`checking for "${handler.openTag}"`);
         if (msg.match(handler.openTag)) {
           // grab everything after opening tag
-          let bits = msg.split(handler.openTag);
-          if (bits.length == 3) {
-            [precontent, containerid, content] = bits;
-          } else {
-            [precontent, content] = bits;
+          let openMatch = msg.match(handler.openTag);
```
Comment (maximusunc) [inline src/output.js:52]: openMatch is never reassigned, so this variable should be a const.

**Review by maximusunc — COMMENTED** [review]
There's a massive empty space right after login that I think is coming from the invmon tags where we aren't removing the newline chars. Not sure if you want to take a look in this PR or make a new issue.

File: src/inventory.js [hunk 2]
```diff
@@ -86,6 +86,7 @@ class Item {
  * @param {string} containerid - the container to which to add the items
  */
 function addItems(self, invdata = "", containerid = "inventory") {
+  invdata = invdata.replace(/^\s+|\s+$/g, "");
```
[the remark this answers is not in the document]
Comment (patrickkwang) [reply src/inventory.js:89]: Including newlines?

[the remark this answers is not in the document]
Comment (patrickkwang) [reply src/inventory.js]: Yes. The trim() documentation really should be more specific about this, and there is a very problematic StackOverflow question (and accepted answer).

**Review by maximusunc — APPROVED** [review]

**Pull request merged**